dev/breeze/src/airflow_breeze/templates/pyproject_TEMPLATE.toml.jinja2

```diff
@@ -69,6 +69,7 @@ classifiers = [
     "Topic :: System :: Monitoring",
 ]
 requires-python = "{{ REQUIRES_PYTHON }}"
+{{ EXTRA_PROJECT_METADATA }}
```

Comment:
aah this is returning `None` in some cases

Reply:
https://github.com/apache/airflow/actions/runs/14474460191/job/40596981411?pr=49306#step:10:6874

Reply:
```suggestion
{% if EXTRA_PROJECT_METADATA %}
{{ EXTRA_PROJECT_METADATA }}
{% endif %}
```

Reply:
I think we need some - there to not generate empty lines

Reply:
yea

Reply:
I hate those `-` in Jinja - it's always trial-and-error to get them right :)

Reply:
Also I pass empty string now for None. Just in case.